src/RuleAsset.php

```diff
@@ -178,6 +181,9 @@ public function getActions()
     {
         $actions                                = parent::getActions();
 
+        $actions['name']['name']              = __('Name');
+        $actions['name']['type']              = 'text';
```

Comment:
```suggestion
```

I'm not in favour (and it's not requested by the customer) of allowing the asset name to be changed. 
The name is also one of the reconciliation keys in the reconciliation rule engine.